elm.json

```diff
@@ -1,39 +1,40 @@
 {
-  "type": "application",
-  "source-directories": [
-    "src"
-  ],
-  "elm-version": "0.19.1",
-  "dependencies": {
-    "direct": {
-      "NoRedInk/elm-json-decode-pipeline": "1.0.0",
-      "elm/browser": "1.0.2",
-      "elm/core": "1.0.4",
-      "elm/html": "1.0.0",
-      "elm/http": "2.0.0",
-      "elm/json": "1.1.3",
-      "elm/regex": "1.0.0",
-      "elm/url": "1.0.0",
-      "hecrj/html-parser": "2.3.4",
-      "krisajenkins/remotedata": "6.0.1",
-      "truqu/elm-base64": "2.0.4"
+    "type": "application",
+    "source-directories": [
+        "src"
+    ],
+    "elm-version": "0.19.1",
+    "dependencies": {
+        "direct": {
+            "NoRedInk/elm-json-decode-pipeline": "1.0.0",
+            "elm/browser": "1.0.2",
+            "elm/core": "1.0.4",
+            "elm/html": "1.0.0",
+            "elm/http": "2.0.0",
+            "elm/json": "1.1.3",
+            "elm/regex": "1.0.0",
+            "elm/url": "1.0.0",
+            "elm-community/basics-extra": "4.1.0",
```

Comment:
this is the package which was added. Probably this file was added by hand before so `elm install` changed it. Let me know if this should be reverted.

Also I only use `uncurry` function from the package - we don't necessary need it - let me know if you want me to remove it. Anyway it provides rather standard lambda calculus style combinatorics missing in elm's Basics module.

Reply:
I tend to avoid adding dependencies (if possible)

Since we are only using one function would it be too much to just copy/paste it (with a comment from where we pasted it)? otherwise we depend on whole basics-extra stuff. 

I find this to work better when maintaining code and also it is much easier to port to new elm versions. I don't have the exact rule but adding a dependency for only one function is a bit of an overkill.

Reply:
> I tend to avoid adding dependencies (if possible)

yeah I'm the same but I learned to make an exception with elm-community stuff. Anyway lets keep this PR focused - I'll remove the dependency and do the uncurryng manually.